tutorials/docs/101_images_feature_extraction.md

```diff
@@ -0,0 +1,51 @@
+# WhizzML: 101 - Images Feature Extraction
+
+Following the schema described in the prediction workflow
+document, this is the code snippet that shows the minimal workflow to
+extract features from images and generate an enriched dataset that can be
+used to train any kind of model.
+
+```
+    ;; step 0: creating a source from the data in your remote
+    ;; "https://github.com/bigmlcom/python/blob/master/data/images/fruits_hist.zip?raw=true" file.
+    ;; The file contains two folders, each
+    ;; of which contains a collection of images. The folder name will be used
+    ;; as label for each image it contains.
+    ;; The source is created enabling image analysis and setting some of the
+    ;; available features (see the API documentation at
+    ;; https://bigml.com/api/sources?id=source-arguments
+    ;; for details). In particular, we extract histogram of gradients and
+    ;; average pixels.
+    (define extracted_features ["average_pixels" "histogram_of_gradients"])
+    (define source-id (create-source
+                        {"remote" "https://github.com/bigmlcom/python/blob/master/data/images/fruits_hist.zip?raw=true"
+                         "image_analysis" {"enabled" true
+                                           "extracted_features" extracted_features}}))
+    ;; When finished, results will be stored and the new ``image_id`` and
+    ;; ``label`` fields will be generated in the source.
+    (log-info "Creating remote source: " source-id)
+    ;; step 1: creating a dataset from the previously created `source` and
+    ;; extracting the requested features
+    (define dataset-id (create-dataset source-id))
```

Comment:
the introduction to this long snippet says nothing about creating anomaly scores, etc... it's supposed to be just feature extraction... perhaps split things in two parts, with a line or two of text saying we're now going to use the features?

Reply:
ok